src/http/HttpClient.js

```diff
@@ -68,15 +68,20 @@ class HttpClient {
     }
 
     /**
-     * Gets a member given an alias.
+     * Gets a member given an alias and a realm.
      *
      * @param {Object} alias - alias to lookup
+     * @param {string} realm - realm of the alias
      * @return {Object} response - response to the API call
      */
-    resolveAlias(alias) {
+    resolveAlias(alias, realm) {
+        if (realm === undefined) {
+            realm = "";
+        }
```

Comment:
can be simplified by doing `resolveAlias(alias, realm='')`, also our style standard is single quotes in JS.

Reply:
fixed